src/assets/app.extensions.json

```diff
@@ -1172,6 +1172,28 @@
         }
       ]
     },
+    "viewerSharedLink": {
```

Comment:
Move it inside "viewer" struct, i.e. "viewer: { shared: { toolbarActions: [] } }"